dashboard/assets/admin/dashboard/js/dropdown_filter.js

```diff
@@ -1,25 +1,42 @@
 var listVisible = false;
+var userMenuVisible = false;
 var listFilter = document.getElementById("changelist-filter");
+var userMenu = document.getElementById("user-menu-options");
 
-function shrink(listFilterElement) {
-  listFilterElement.setAttribute("style", "height:54px");
-  listFilterElement.style.overflow = "hidden";
-  listVisible = false;
+function shrink(dropDownElement, height) {
+  var heightStyle = "height:" + String(height) + "px";
+  dropDownElement.setAttribute("style", heightStyle);
+  dropDownElement.style.overflow = "hidden";
 }
 
-function expand(listFilterElement) {
-  listFilterElement.setAttribute("style", "height:auto");
-  listFilterElement.style.overflow = "visible";
-  listVisible = true;
+function expand(dropDownElement) {
+  dropDownElement.setAttribute("style", "height:auto");
+  dropDownElement.style.overflow = "visible";
```

Comment:
would be better to just have a css class then you could just toggle wether the element has the class or not

Something like:

```
if (element.hasClass('expanded')) {
    element.removeClass('expanded');
} else {
    element.addClass('expanded');
}
```